features/steps/when.py

```
@@ -16,6 +16,11 @@ def run(context: Context, command: str):
         # Override the HOME variable
         env["HOME"] = context.home_dir
 
+    # if os.name == "nt" and not "COMSPEC" in os.environ:
+    os.environ["COMSPEC"] = "C:\\Windows\\System32\\cmd.exe"
+    os.environ["ComSpec"] = "C:\\Windows\\System32\\cmd.exe"
```

Ruff (0.8.2)

Use capitalized environment variable `COMSPEC` instead of `ComSpec`

Replace `ComSpec` with `COMSPEC`

(SIM112)